azure-pipelines.yml

```diff
@@ -158,8 +126,8 @@ stages:
            # add to ld_lib path (rpaths are missing?)
            export LD_LIBRARY_PATH=${VTK_LIB_DIR}:${OSPRAY_LIB_DIR}:${QT_LIB_DIR}
            # run test suite on silo + blueprint tests
-           export TESTS="$(python_test_files)"
-           ./run_visit_test_suite.sh --fuzzy --pixdiff 10 --avgdiff 10 --numdiff 0.1 -v ${TESTS}
+           export TESTS="$(find ../../src/test/tests -name '*.py' -exec grep -H -m 1 CLASSES {} \; | grep prci | cut -d':' -f1)"
```

Comment:
I am not familiar enough with all of these commands to know exactly what is happening... if someone names a variable in a python test `prci` will the test get unwittingly triggered in CI?